atomate/vasp/firetasks/parse_outputs.py

```diff
@@ -708,6 +708,134 @@ def run_task(self, fw_spec):
         logger.info("Thermal expansion coefficient calculation complete.")
 
 
+@explicit_serialize
+class PolarizationToDb(FiretaskBase):
+    """
+    Recovers the same branch polarization and spontaneous polarization
+    for a ferroelectric workflow.
+    """
+
+    optional_params = ["db_file"]
+
+    def run_task(self, fw_spec):
+        import re
+        from pymatgen.analysis.ferroelectricity.polarization import Polarization, get_total_ionic_dipole, \
+            EnergyTrend
+
+        wfid = list(filter(lambda x: 'wfid' in x, fw_spec['tags'])).pop()
+        db_file = env_chk(self.get("db_file"), fw_spec)
+        vaspdb = VaspCalcDb.from_db_file(db_file, admin=True)
+
+        # ferroelectric workflow groups calculations by generated wfid tag
+        polarization_tasks = vaspdb.collection.find({"tags": wfid, "task_label": {"$regex": ".*polarization"}})
+
+        tasks = []
+        outcars = []
+        structure_dicts = []
+        sort_weight = []
+        energies_per_atom = []
+        energies = []
+        zval_dicts = []
+
+        for p in polarization_tasks:
+            # Grad data from each polarization task
+            c = p['calcs_reversed'][0]
```

Comment:
can you avoid creating these shortcuts? impossible to remember what they refer to ...

Reply:
Done.